### PR TITLE
[psc-ide] Better logging and diagnostics

### PR DESCRIPTION
**Pull request by kRITZCREEK**
Adds checks for Externsfile versions and a few simple measurements for debugging/benchmarking.

### CODE REVIEW

**Comment by paf31** [comment]
👍 Looks good!

**Comment by kRITZCREEK** [comment]
Thanks 👍

File: psc-ide-server/README.md [hunk 1]
```diff
@@ -22,9 +22,11 @@ It supports the following options:
 - `--output-directory`: Specify where to look for compiled output inside your
   project directory. Defaults to `output/`, relative to either the current
   directory or the directory specified by `-d`.
-- `--debug`: Enables some logging meant for debugging
+<<<<<<< HEAD
```
Comment (LiamGoodacre) [inline psc-ide-server/README.md]: Not sure this was supposed to go in 😄

Comment (kRITZCREEK) [inline psc-ide-server/README.md]: whoops :D

**Pull request merged**